pulp_rpm/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '3.4.0b1.dev'
+__version__ = '3.4.0.dev'
```

Comment:
is it an equivalent of 3.4.0.a1.dev?
I'm trying to figure out what would be the result of this comparison:
3.4.0b1.dev < 3.4.0.dev

Reply:
i have no idea how that would compare. are you wondering about the bindings versions? what is the concern?

Reply:
yes, the bindings. Just in case, I have travis issues for a while and release won;t happen fast, it can be a bindings problem but for a short time.

Reply:
I had to change bump2version on pulp_ansible, I can fix it. RPM always use this `b1`?
https://github.com/pulp/pulp_ansible/blob/master/.bumpversion.cfg#L6-L8